ktor-utils/common/src/io/ktor/util/Annotations.kt

```diff
@@ -34,7 +34,7 @@ annotation class InternalAPI
 @RequiresOptIn(
     level = RequiresOptIn.Level.WARNING,
     message = "This API is experimental. " +
-        "It could be removed or changed in future releases or it's behaviour may be different."
+        "It could be removed or changed in future releases or its behaviour may be different."
```

Comment:
Should be 

"It cold be removed or changed in future release, or its behaviour may be different."